intellij-plugin/.gitignore

```diff
@@ -0,0 +1 @@
+mockJDK
```

Comment:
I never know how multiple .gitignore play together... Can we put this in the main one?